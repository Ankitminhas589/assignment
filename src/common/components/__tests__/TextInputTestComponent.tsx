import React from 'react';
import {fireEvent, render} from '@testing-library/react-native';
import CurrencyTextInputComponent from '../TextInputComponent';
describe('Image with loader tests', () => {
  it('renders correcly', async () => {
    const {toJSON, getByTestId} = render(
      <CurrencyTextInputComponent value="" onValueChange={() => {}}></CurrencyTextInputComponent>,
    );
    expect(toJSON()).toMatchSnapshot();
  });
});
