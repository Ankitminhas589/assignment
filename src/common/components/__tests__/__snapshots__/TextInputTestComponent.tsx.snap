// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image with loader tests renders correcly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "borderBottomWidth": 0.5,
      "borderColor": "#a7a7a7",
      "flexDirection": "row",
      "paddingVertical": 20,
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#04d167",
        "borderRadius": 8,
        "height": 40,
        "justifyContent": "center",
        "width": 70,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#ffffff",
          "fontSize": 24,
          "fontWeight": "bold",
        }
      }
    >
      S$
    </Text>
  </View>
  <TextInput
    keyboardType="numeric"
    onChangeText={[Function]}
    style={
      Object {
        "flex": 1,
        "marginLeft": 30,
      }
    }
    value=""
  />
</View>
`;
